packages/aragon-wrapper/src/rpc/handlers/events.js

```diff
@@ -1,8 +1,7 @@
 export default function (request, proxy) {
-  const fromBlock = request.params && request.params[0]
-
-  if (fromBlock) {
-    return proxy.events(null, { fromBlock })
+  const [eventsOptions] = request.params
+  if (eventsOptions != null) {
```

Comment:
Was the loose comparison deliberate to get undefined too?

Reply:
Yes, in case that last param wasn't given it would be undefined.